operators.py

```diff
@@ -0,0 +1,52 @@
+import yaml
+from interpolation import VerticalGrid
+import parser
+
+with open("config.yml", "r", encoding="utf8") as f:
+    config = yaml.safe_load(f)
+
+
+def get_overlapping_pixels(sat_lat, sat_lon, gc_lat, gc_lon):
+    pass
+    # calculate pixels which overlap each satellite observation
+    # assumes model pixel size >> satellite pixel size
+
+def get_closest_time(gc_time, sat_time):
+    pass
+    # get closest times in model to satellite times
+
+def get_satellite_column(
```

Comment:
So this applies the mapping to the GEOS-Chem data? Maybe we should rename it, because this makes me think this function only uses satellite data, and not model data.

Reply:
changed to get_model_columns